src/applications/personalization/profile-2/components/PersonalInformation.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
 
-const PersonalInformation = () => <h1>Personal Information</h1>;
+const PersonalInformation = () => <h2>Personal Information</h2>;
```

Comment:
If your design spec requires `h1` styling you could add the class `vads-u-font-size--h1`! 🙂

Reply:
This is/was just a placeholder that I wanted to make sure nested properly.